src/pages/List.js

```diff
@@ -269,83 +269,110 @@ export default function List({ token }) {
 
   return (
     <>
-      <h1>This Is Your Grocery List</h1>
-      <h2>It uses the token: {token}</h2>
-      <label htmlFor="thesearch">Search Grocery List Items </label>
-      <div className="flex">
-        <input
-          type="text"
-          placeholder="enter grocery item"
-          value={query}
-          id="thesearch"
-          onChange={(e) => setQuery(e.target.value)}
-        />
-        <IconButton
-          onClick={handleReset}
-          icon={
+      <h1 className="mt-5 mb-10 text-3xl self-start font-light">
+        Things I'll need
+      </h1>
+      <div
+        class="bg-teal border-t-4 rounded-b text-midnight-green border-caribbean-green px-4 py-3 shadow-md w-full"
+        role="alert"
+      >
+        <div class="flex">
+          <div class="py-1 text-caribbean-green">
```

Comment:
Looks good! A lot like the Figma design!

Just a quick reminder to change `class` to `className`!

Reply:
Will do! 👍